Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -7595,6 +7595,19 @@ def replace_str(src_str):
 # ############################## REGEX FORMATTING end ###############################
 
 
+def is_filename_valid(filename):
+    """
+    Checking if the file name contains invalid characters.
+
+    :param filename: The file name
+    :type filename: str
+
+    :return: None
+    """
+    if not re.match(r"^[^~)('\\!*<>:;,?\"*|/]+$", filename):
+        raise DemistoException('The file name contains invalid characters')
```

Comment:
Why raise an exception and not return a boolean?
That way you could easily use it in `if` statements,
And the param might not always be called `filename`.